app/guid-file/template.hbs

```diff
@@ -146,57 +148,57 @@
                                     {{t 'general.save'}}
                                 </Button>
                             {{else}}
-                                <div>{{t 'file_detail.title'}}</div>
+                                <h3>{{t 'file_detail.title'}}</h3>
```

Comment:
Channeling my inner @futa-ikeda, I wonder if it would make this more semantic to turn these into description lists instead, like we did with the [node metadata form](https://github.com/CenterForOpenScience/ember-osf-web/blob/upstream/guid-metadata/lib/osf-components/addon/components/node-metadata-form/template.hbs#L173-L184).

Reply:
Ok, let me try it out.

Reply:
Ok, these elements are now update with data list, term and description elements.